examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java

```diff
@@ -494,7 +494,7 @@ public static void main(String[] args) throws IOException {
                .to(tableRef)
                .withSchema(FormatForBigquery.getSchema())
                .withCreateDisposition(BigQueryIO.Write.CreateDisposition.CREATE_IF_NEEDED)
-               .withWriteDisposition(BigQueryIO.Write.WriteDisposition.WRITE_TRUNCATE));
+               .withWriteDisposition(BigQueryIO.Write.WriteDisposition.WRITE_APPEND));
```

Comment:
Consider leaving this as WRITE_TRUNCATE for streaming:

```
.withWriteDisposition(options.isStreaming()
    ? BigQueryIO.Write.WriteDisposition.WRITE_APPEND
    : BigQueryIO.Write.WriteDisposition.WRITE_TRUNCATE);
```

Also, fix the NullPointerException below...